shared/rest/src/main/java/io/pravega/shared/rest/impl/RESTServerConfigImpl.java

```diff
@@ -22,24 +22,34 @@
 import lombok.Builder;
 import lombok.Getter;
 
+import java.util.Arrays;
 import java.util.Properties;
 
 /**
  * REST server config.
  */
 @Getter
+@Builder
 public class RESTServerConfigImpl implements RESTServerConfig {
     private final String host;
     private final int port;
     private final boolean authorizationEnabled;
     private final String userPasswordFile;
     private final boolean tlsEnabled;
+    private final String[] tlsProtocolVersion;
     private final String keyFilePath;
     private final String keyFilePasswordPath;
 
-    @Builder
+    public static final class RESTServerConfigImplBuilder {
+        private String[] tlsProtocolVersion = new String[] {"TLSv1.2", "TLSv1.3"};
```

Comment:
1. `final`
2. Why is this hardcoded?
3. Why not use `TlsProtocolVersion` class here?

Reply:
1. If it's final makes the @builder annotation fail as a value cannot be assigned to a final variable.
2.  Because import Config.TLS_PROTOCOL_VERSION causes a dependency of Controller in Shared package.
3. TlsProtocolVersion is just a utility class to parse and validate the values.